test/e2e/helpers.sh

```diff
@@ -15,4 +15,9 @@ function shp () {
 	fi
 
 	${BIN} ${*}
-}
\ No newline at end of file
+}
+
+function randomDnsSubdomainName () {
```

Comment:
What about `random_name` instead? I think it would be more descriptive considering it's employed as the actual build name.

Reply:
fixed